package.json

```diff
@@ -75,7 +75,7 @@
     "zuul": "^3.12.0"
   },
   "engines": {
-    "node": ">=6.4.0 !13"
+    "node": ">=6.4.0 || <13 >=14"
```

Comment:
This is not correct; you want the `<13` to be before the `||`. 

```
$ node
Welcome to Node.js v16.2.0.
Type ".help" for more information.
> const {satisfies} = require('semver')
undefined
> satisfies('v16.1.0', ">=6.4.0 || <13 >=14")
true
> satisfies('v13.1.0', ">=6.4.0 || <13 >=14")
true
> satisfies('v13.1.0', ">=6.4.0 <13 || >=14")
false
> satisfies('v16.1.0', ">=6.4.0 <13 || >=14")
true
```

Reply:
I've opened https://github.com/visionmedia/superagent/pull/1666 to make it easier for the maintainer to merge and release.